### DISCUSSION TITLE
[Question] How to decoration clip on chart

### DISCUSSION THREAD

**Discussion by Chung-Chase52** [patrykandpatrick/vico#295] [Ideas]
I want to create something like the first image.

However, when it exceeds the chart area, it doesn't get drawn completely.  (Second Image)

How can I draw the circles on both sides entirely?

[I Wish]
![스크린샷 2023-06-12 오전 11 40 19](https://github.com/patrykandpatrick/vico/assets/66232574/4528d11a-2c98-4164-91a9-010bbbe34f80)

[Current Outcome]
![스크린샷 2023-06-12 오전 11 40 19 복사본](https://github.com/patrykandpatrick/vico/assets/66232574/afe5d4ac-a9eb-4435-aed7-9bdbcfbcec53)

**Comment by patrickmichalik** [comment]
Hello! I’m afraid this isn’t supported at the moment, but we’ll be looking into making it possible.

**Comment by Gowsky** [comment]
Hi @Chung-Chase52, [Vico 2.0.0-alpha.23](https://patrykandpatrick.com/vico/releases/2.0.0-alpha.23/) makes this possible. Please [see here](https://patrykandpatrick.com/vico/releases/2.0.0-alpha.23/#miscellaneous).

**Comment by Gowsky** [comment]
[Vico 2.0.0](https://github.com/patrykandpatrick/vico/releases/tag/v2.0.0) is now available.
